pkgs/tools/security/prs/default.nix

```diff
@@ -31,10 +32,16 @@ rustPlatform.buildRustPackage rec {
     done
   '';
 
-  nativeBuildInputs = [ gpgme pkg-config python3 ];
+  nativeBuildInputs = [ gpgme installShellFiles pkg-config python3 ];
 
   buildInputs = [ dbus glib gpgme gtk3 libxcb ];
 
+  postInstall = ''
+    for shell in bash fish zsh; do
+      installShellCompletion --cmd prs --$shell <($out/bin/prs internal completions $shell --stdout)
```

Comment:
@lilyball This generates `share/fish/vendor_completions.d/prs.fish` while it should be `share/fish/vendor_completions.d/_prs`.